Source/WebKit/GPUProcess/graphics/RemoteImageBufferSet.h

```diff
@@ -80,7 +80,10 @@ class RemoteImageBufferSet : public IPC::StreamMessageReceiver, public ImageBuff
 
     bool isOpaque() const
     {
+#if HAVE(IOSURFACE_RGB10)
         return m_pixelFormat == WebCore::ImageBufferPixelFormat::RGB10 || m_pixelFormat == WebCore::ImageBufferPixelFormat::BGRX8;
+#endif
+        return m_pixelFormat == WebCore::ImageBufferPixelFormat::BGRX8;
```

Comment:
I think two return statements in a function when `HAVE(IOSURFACE_RGB10)` is true is a little confusing. I think you either add `#else` and include the second return in it:

```
#if HAVE(IOSURFACE_RGB10)
        return m_pixelFormat == WebCore::ImageBufferPixelFormat::RGB10 || m_pixelFormat == WebCore::ImageBufferPixelFormat::BGRX8;
#else
        return m_pixelFormat == WebCore::ImageBufferPixelFormat::BGRX8;
#endif
```

Or remove the duplicate check:

```
#if HAVE(IOSURFACE_RGB10)
        if (m_pixelFormat == WebCore::ImageBufferPixelFormat::RGB10)
            return true;
#endif
        return m_pixelFormat == WebCore::ImageBufferPixelFormat::BGRX8;
```

I prefer the later.